configs/opencv-ovms/envs/gst.env

```diff
@@ -1,4 +1,5 @@
 COLOR_WIDTH=1920
 COLOR_HEIGHT=1080
 COLOR_FRAMERATE=15
-OCR_SPECIFIED=5
\ No newline at end of file
+OCR_SPECIFIED=5
+PIPELINE_SCRIPT="yolov5s.sh"
```

Comment:
we probably want to seek other way for this...this env opens a door way for malicious script injection and it is very bad for security reason: we can do this in another issue or PR.